admin/spec/features/spree/admin/orders/edit_order_spec.rb

```
@@ -57,12 +58,12 @@
         click_on 'dropdown-toggle'
         click_on Spree.t('admin.edit_shipping_address')
       end
-      wait_for_turbo
+      wait_for_dialog
 
-      tom_select address_to_select.to_s.gsub('<br/>', ", "), from: Spree.t(:existing_address)
-      wait_for_turbo
+      select address_to_select.to_s.gsub('<br/>', ", "), from: 'shipping_address_id'
```

RuboCop (1.81.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)